PowerPointLabs/PowerPointLabs/Utils/Graphics.cs

```diff
@@ -79,15 +79,23 @@ public static void ExportShape(ShapeRange shapeRange, string exportPath)
                               slideHeight, PpExportMode.ppScaleToFit);
         }
 
-        public static Image ShapeToImage(Shape shape)
+        public static Bitmap ShapeToImage(Shape shape)

```

Comment:
it's rare the there can be race condition here, do u want to lock/sync this method?